crates/bevy_time/src/timer.rs

```diff
@@ -343,6 +343,38 @@ impl Timer {
         1.0 - self.percent()
     }
 
+    /// Returns the remaining time in seconds
+    ///
+    /// # Examples
+    /// ```
+    /// # use bevy_time::*;
+    /// use std::cmp::Ordering;
+    /// use std::time::Duration;
+    /// let mut timer = Timer::from_seconds(2.0, false);
+    /// timer.tick(Duration::from_secs_f32(0.5));
+    /// let result = timer.remaining_secs().total_cmp(&1.5);
+    /// assert_eq!(Ordering::Equal, result);
+    /// ```
+    #[inline]
+    pub fn remaining_secs(&self) -> f32 {
+        self.duration().as_secs_f32() - self.elapsed().as_secs_f32()
```

Comment:
I think it probably makes sense to make this `self.remaining().as_secs_f32()`. That way we don't re-specify the "remaining" logic in multiple places, and we only do a single float conversion at the end (which might preserve precision in some cases).

Reply:
I totally agree, going to change it